dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/BaseControls/Auxiliaries/GridProtection.mo

```diff
@@ -85,6 +88,18 @@ model GridProtection "Grid protection system for wind turbines (IEC N°61400-27-
     Dialog(tab = "Operating point"));
 
 equation
+  when greater.y then
```

Comment:
when greater.y and not pre(greater.y)?

Reply:
As it is a when, doesn't it natively encompass such a condition, ie the idea that you have a transition?
I don't remember if there are cases in which we need to have the pre()

Reply:
I'm not sure anymore :) do we have a testcase in which one of those values become true?

Reply:
[TEST2.zip](https://github.com/dynawo/dynawo/files/15010232/TEST2.zip)
I'm using this test case to test this feature

Reply:
Je vais faire 2-3 tests pour voir si le pre est necessaire. En attendant est ce qu'on pourrait ajouter ce cas test aux nrt de dynawo-rte ? merci :)

Reply:
Should I also include the obtained timeline in the reference outputs ?

Reply:
yes please :)